editor/plugins/script_text_editor.cpp

```diff
@@ -637,6 +644,17 @@ void ScriptTextEditor::_update_warnings() {
 		warnings_panel->pop(); // Cell.
 	}
 	warnings_panel->pop(); // Table.
+	if (warning_line_color.a != 0.0) {
+		CodeEdit *te = code_editor->get_text_editor();
+		for (int i = 0; i < te->get_line_count(); i++) {
+			for (const ScriptLanguage::Warning &W : warnings) {
```

Comment:
[saw this was already approved, definitely not a blocker]

I feel like it most cases it would be better to loop over the warnings and only mark the ranges indicated by each one. technically the same big O but in general I would expect several 1 line warnings and potentially 100s or 1000s of lines of code. The current code would be better as there were more warnings and also many overlapping the same lines.
```suggestion
		for (const ScriptLanguage::Warning &W : warnings) {
			for (int i = W.start_line - 1; i < W.end_line; i++) {
```